### PR TITLE
Add Feathers JS example

### PR DESCRIPTION
**Pull request by havenS**
This example demonstrates how *Next* can be used in combination with *Feathers JS*.

### CODE REVIEW

File: examples/with-feathersjs/package.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  "contributors": [],
+  "bugs": {},
+  "engines": {
+    "node": ">= 0.12.0"
+  },
+  "scripts": {
+    "test": "npm run jshint && npm run mocha",
+    "jshint": "jshint src/. test/. --config",
+    ".prestart": "next build",
+    "start": "node src/",
```
Comment (arunoda) [inline examples/with-feathersjs/package.json]: We recommend to run feathers in a different server. That's the way we need to recommend to create apps.

Use next just for serving the app.
Is it possible to change this into that?

Comment (havenS) [inline examples/with-feathersjs/package.json]: Thanks @arunoda, I understand and I wondered if the example was even relevant. But for small applications, simple React SSR in Feathers JS and if we want to keep everything in the same place, I think it makes sense.
If Feathers has to be run on a different server, this example is thus irrelevant as no specific setup is needed.

Comment (reel) [inline examples/with-feathersjs/package.json]: You might want to use https://www.npmjs.com/package/henri for that

**Comment by rauchg** [comment]
In the spirit of not trying to confuse people (per @arunoda's comments), and due to the larger nature of this example, I'd like to *link to this* from the Wiki, as a question in the FAQ. Thanks for your contribution though!

**Pull request closed without merging**